django_simple_bulma/extensions/bulma-dropdown/dist/js/bulma-dropdown.js

```diff
@@ -0,0 +1,40 @@
+(function() {
+    window.dropdowns = {};
+
+    function handle_event(e) {
+        show();
+
+        function clickEvent() {
+            hide();
+            document.body.removeEventListener("click", clickEvent);
+        };
+
+        document.body.addEventListener("click", clickEvent);
+
```

Comment:
This newline is satanic.
```suggestion
```

Reply:
There really is nowhere for me to exercise my devilish coding practices 🙄